lib/daru/view/adapters/googlecharts/table_chart_common_methods.rb

```diff
@@ -0,0 +1,68 @@
+require 'google_visualr'
```

Comment:
There is already one module is added for display purpose , please [check](https://github.com/SciRuby/daru-view/blob/master/lib/daru/view/adapters/googlecharts/display.rb#L94) display module.
Isn't it display module contains similar methods?

Reply:
I have moved the common methods to display module. Sorry, I got confused. Actually, display module contained methods that called the methods of base_chart.rb and data_table.rb (like to_js and others) and there was no method that was called back from these files to display.